lib/ui/widgets/topics_widget.dart

```diff
@@ -41,21 +41,24 @@ class TopicsWidget extends StatelessWidget {
 
 class TopicsButtonWidget extends StatefulWidget {
   final String topic;
+  final bool canPress;
 
-  TopicsButtonWidget({Key key, @required this.topic}) : super(key: key);
+  TopicsButtonWidget({Key key, @required this.topic, this.canPress})
```

Comment:
```suggestion
  TopicsButtonWidget({Key key, @required this.topic, this.canPress = false})
```

Reply:
To fix:

```bash
The following assertion was thrown while handling a gesture:
Failed assertion: boolean expression must not be null

When the exception was thrown, this was the stack: 
#0      _TopicsButtonWidgetState.build.<anonymous closure> (package:voting_app/ui/widgets/topics_widget.dart:106:20)
#1      _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:993:19)
#2      _InkResponseState.build.<anonymous closure> (package:flutter/src/material/ink_well.dart:1111:38)
#3      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:183:24)
#4      TapGestureRecognizer.handleTapUp (package:flutter/src/gestures/tap.dart:598:11)
```

When a button is pressed. The default value is `this.canPress = false`